pages/audit/overview.mdx

```diff
@@ -10,6 +10,8 @@ The Avo tracking plan audit reviews your event and property definitions and flag
 
 The tracking plan audit monitors every change you make in your Avo Tracking Plan and highlights if new issues arise. You can think about the tracking plan audit like a spell-checker for your tracking plan.
 
+Furthermore, all changes on branches are also checked in the <Link passHref href="/audit/branch-audit"><a>Branch Audit</a></Link> to see if you have introduced any new issues on your branch.
```

Comment:
Unrelated to your changes specifically, but we are really missing a picture here of the tracking plan audit in the sidebar so people know what we are talking about 🤔